src/icon-tab-bar.scss

```diff
@@ -307,10 +322,19 @@ $fd-icon-tab-bar-semantic-values: (
     }
 
     &.#{$block}__item--overflow {
-      @include fd-reset-margins-x();
+      position: absolute;
+      top: 50%;
+      transform: translateY(-50%);
+      margin: 0;
 
-      flex: 1;
-      justify-content: flex-end;
+      &-left {
+        left: 0;
+
+        @include fd-rtl() {
+          left: auto;
+          right: 0;
+        }
```

Comment:
Forgot to say during the previous review that there's a mixin for setting the position:
<img width="430" alt="Screen Shot 2021-08-05 at 9 50 11 AM" src="https://user-images.githubusercontent.com/39598672/128363275-7627fc20-ba70-449e-9b7b-2620596624c5.png">

Will handle the rtl

Reply:
We have mixins for any cases of life)